client/src/Component/Pages/AddPicture/index.js

```diff
@@ -0,0 +1,106 @@
+import React from "react";
+
+import Header from "../../CommonComponent/Header";
+import Footer from "../../CommonComponent/Footer";
+import Button from "../../CommonComponent/Button";
+import Input from "../../CommonComponent/Input";
+
+import "./style.css";
+
+export default function(props) {
+  const [title, setTitle] = React.useState("");
+  const [file, setFile] = React.useState("");
+  const [error, SetError] = React.useState(false);
+  const [typeError, SetTypeError] = React.useState(false);
+  const [fetchError, SetfetchError] = React.useState(false);
+  const [buttonContent, setbuttonContent] = React.useState("Save");
+  const acceptedTypes = ["jpeg", "jpg", "gif", "png"];
+  let type = file.split(".")[1];
```

Comment:
the name of the file can contain also dot like `caty.cat.jpeg`
in your code, the type will be `cat` but this is wrong
you can fix it by 
```suggestion
  let type = file.split(".")[file.split('.').length-1];
```